pkg/bundle/internal/target/target.go

```diff
@@ -396,3 +397,21 @@ func (b *Data) Populate(pool *util.CertPool, formats *trustapi.AdditionalFormats
 	}
 	return nil
 }
+
+func TruststoreHash(data []byte, bundle *trustapi.AdditionalFormats) string {
```

Comment:
Now that you suggest using this function in tests, I think we need a unit test of the function. I believe it is now possible to change the function behavior without breaking the tests.

Reply:
I added a unit test.